### ISSUE TITLE
Issue converting labelme JSON to YOLODataset

### ISSUE THREAD

**Issue by zaheerkzz** [rooneysh/Labelme2YOLO#13]
Here is my _labelme_ **JSON**:

`{
    "version": "5.3.1",
    "flags": {},
    "shapes": [
        {
            "label": "Center",
            "points": [
                [
                    1400,
                    900
                ],
                [
                    1500,
                    900
                ],
                [
                    1500,
                    1000
                ],
                [
                    1400,
                    1000
                ]
            ],
            "group_id": null,
            "description": "",
            "shape_type": "Rectangle",
            "flags": {}
        },
        {
            "label": "LeftGaurd",
            "points": [
                [
                    1320,
                    920
                ],
                [
                    1420,
                    920
                ],
                [
                    1420,
                    1020
                ],
                [
                    1320,
                    1020
                ]
            ],
            "group_id": null,
            "description": "",
            "shape_type": "Circle",
            "flags": {}
        },
        {
            "label": "LeftTackle",
            "points": [
                [
                    1240,
                    920
                ],
                [
                    1340,
                    920
                ],
                [
                    1340,
                    1020
                ],
                [
                    1240,
                    1020
                ]
            ],
            "group_id": null,
            "description": "",
            "shape_type": "Circle",
            "flags": {}
        }
    ],
    "imagePath": "img/latest-00-QBShotGun-WRLeftHashmarkOn-WRRightHashmarkOn-FeatherLeft-WRRightHashmarkOff-FeatherRight-normal-16686378232057195.png",
    "imageData": "",
    "imageHeight": 1200,
    "imageWidth": 2500
}`

Im trying to convert to **YOLO** Dataset using this command:
`labelme2yolo --json_dir json --val_size 0.15 --test_size 0.15`

It Generates a Folders **YOLODataset** -> **images, label** -> **test, train val**, but the folders are empty.

Im using:

```
Python 3.10.12
YoloV8
labelme2yolo 0.1.2 
```

**Comment by rooneysh** [comment]
Is there any exception info?

**Comment by zaheerkzz** [comment]
The issue fixed on Linux. 
On windows this is the exception info.

raise QtBindingsNotFoundError from None
qtpy.QtBindingsNotFoundError: No Qt bindings could be found

I already installed all requirements.

**Comment by sudhin05** [comment]
The issue arises due to 

> imageData": ""

you could use the fix as suggested in the PR #21

**Comment by PengPeng-JunJun** [comment]
@zaheerkzz Have you slove this problem?I have do like the codes successful.We can discuss it

**Comment by zaheerkzz** [comment]
Yes, it's solved> @zaheerkzz Have you slove this problem?I have do like the codes successful.We can discuss it

**Comment by zaheerkzz** [comment]
Yes, it's solved> @zaheerkzz Have you slove this problem?I have do like the codes successful.We can discuss it

**Comment by zaheerkzz** [comment]
Yes, it's solved> @zaheerkzz Have you slove this problem?I have do like the codes successful.We can discuss it

**Comment by zaheerkzz** [comment]
Yes, it's solved> @zaheerkzz Have you slove this problem?I have do like the codes successful.We can discuss it

**Comment by PengPeng-JunJun** [comment]
Good job
